mavros_msgs/msg/ActuatorControl.msg

```diff
@@ -10,6 +10,7 @@ uint8 PX4_MIX_PAYLOAD = 2
 uint8 PX4_MIX_MANUAL_PASSTHROUGH = 3
 #uint8 PX4_MIX_FC_MC_VIRT = 4
 #uint8 PX4_MIX_FC_FW_VIRT = 5
+uint8 PX4_DIRECT_MOTOR_CONTROL = 99
```

Comment:
Perhaps better to keep `PX4_MIX_` prefix.

```suggestion
uint8 PX4_MIX_DIRECT_MOTOR_CONTROL = 99
```

Reply:
Well it really doesn't go through the mixer, thats why I kept it without the _MIX_ . 

Yes, I also was puzzled with that, because on the mavlink its group_mlx as well.